.gitignore

```diff
@@ -2,3 +2,5 @@ tmp/
 spec/decidim_dummy_app/
 .rspec-failures
 .byebug_history
+.ruby-version
+Gemfile.lock
```

Comment:
I'm sorry I prefer to have both files checked in the repository

Reply:
Done